utils/outputwriter/standardoutput.go

```diff
@@ -5,7 +5,6 @@ import (
 	"strings"
 
 	"github.com/jfrog/froggit-go/vcsutils"
-	"github.com/jfrog/jfrog-cli-core/v2/xray/formats"
 )
 
 type StandardOutput struct {
```

Comment:
The "StandardOutput" and the "SimplifiedOutput" can inherit from the same struct. That will help to remove redundancy in variables and methods.